pandas/_libs/interval.pyx

```diff
@@ -40,9 +41,9 @@ cdef class IntervalMixin:
 
 
 cdef _interval_like(other):
-    return (hasattr(other, 'left')
-            and hasattr(other, 'right')
-            and hasattr(other, 'closed'))
+    return (hasattr(other, 'left') and
+            hasattr(other, 'right') and
+            hasattr(other, 'closed'))
```

Comment:
Can we keep the original code and add this to the ignored ones (it's error code W503)? 
I much prefer having the 'and' on the new line, and the most recent PEP8 agrees with me (https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator)

Reply:
Good call!

Reply:
Thanks for adding it to the ignore list. 
Can you undo the changes above? (and there are some below as well)

Reply:
Will do.